utils/soca/gdas_postprocincr.h

```diff
@@ -281,7 +281,7 @@ class PostProcIncr {
     oops::Log::info() << "======      Quality control on increment" << std::endl;
 
     // Perform quality control on the increment
-    gdasapp::qcIncrement::qcIncrement(xb, dx, config, geom);
+    gdasapp::incrqc::qcIncrement(xb, dx, config, geom);
```

Comment:
As suggested by @shlyaeva : Since this method will hide bugs or configuration issues in the DA, add an option to skip the QC here.